src/main/java/org/spongepowered/api/command/args/GenericArguments.java

```diff
@@ -1432,6 +1432,16 @@ protected Object parseValue(CommandSource source, CommandArgs args) throws Argum
             }
         }
 
+        @Override
+        public List<String> complete(CommandSource src, CommandArgs args, CommandContext context) {
```

Comment:
No. You're completely changing the completion of this element - this removes the ability to use regex to complete, _and_ you are removing the ability to complete a selector.

See 

https://github.com/SpongePowered/SpongeAPI/blob/a3e2b07a0de0877037eb16091425ee9757649c57/src/main/java/org/spongepowered/api/command/args/SelectorCommandElement.java#L61-L71

and

https://github.com/SpongePowered/SpongeAPI/blob/a3e2b07a0de0877037eb16091425ee9757649c57/src/main/java/org/spongepowered/api/command/args/PatternMatchingCommandElement.java#L76-L83

for what this should be doing.

Honestly, this is masking a problem I think. If it's a problem with the player element then I'm pretty sure it'll be a problem elsewhere and there might be a bug with the completion code somewhere. Overriding the complete method in the player element is _not_ the way to do it.